src/styles/default-theme.css

```diff
@@ -1,4 +1,5 @@
 :root {
+  --cc-ff-form-legend: 'Source Sans 3';
```

Comment:
issue: we need a default fallback

```
--cc-ff-form-legend: 'Source Sans 3', sans-serif;
```

I wonder if we should just put `sans-serif` and let the console customize it.